app/models/user.rb

```diff
@@ -712,7 +712,7 @@ def cartodb_avatar
   end
 
   def avatar
-    self.avatar_url.nil? ? "//#{self.default_avatar}" : "http://" + self.avatar_url
+    self.avatar_url.nil? ? "//#{self.default_avatar}" : self.avatar_url
```

Comment:
What did this produce before?

Reply:
Ironically the avatar image doesn't work properly with or without this change.  However I believe it's primarily because we don't deploy the default avatar assets as part of Bloomberg build.

Reply:
Then I would go for disabling the tests as not relevant instead of changing the code base! It's very minor in this case, but I think it's the right thing to do.

What do you think?

Reply:
The test is still valid since it verifies that ::Carto::User matches ::User (for all properties).  Arguably it caught a real issue since there is a mismatch between this file and https://github.com/cartodb-org/cartodb/blob/68c33299886bf2ccebee6248fa14d75193ece81e/app/models/carto/user.rb#L146.  

An alternative is to modify the unit test to not test the avatar property in Bloomberg's case

Reply:
If you're not using the property, disabling the test that checks that property (with the pending reason explaining why) is the way to go IMO.

Reply:
@gfiorav with this change we will be back to compatibility with original repo. future merges will not worry about conflict/diff

Reply:
@mbektas thanks for clarifying that! I hadn't understood this is upstream code. If that's the case, then all 👍 from my side